spec/models/feedback_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Feedback, type: :model do
```

Comment:
estaría bueno agregar los tests para:
- validar que pertenezca a `receiver`
- validar la presencia del atributo

acá puedes encontrar cómo https://github.com/thoughtbot/shoulda-matchers#should-vs-is_expectedto